freshmaker/config.py

```diff
@@ -439,6 +439,11 @@ class Config(object):
             'default': '',
             'desc': 'Root url of Flatpak index service'
         },
+        'hidden_content_set_pattern': {
```

Comment:
This is not pattern but suffix.

In the configuration, using a regex pattern seems like a better more flexible solution: `exclude_content_sets_pattern`

Reply:
Ack. Fixing.